### PR TITLE
MTM-43356: added info about extra indicator row

### PR DESCRIPTION
**Pull request by kaczynskid-sag**
added info about extra indicator row in case of export limit is reached

### CODE REVIEW

**Review by c8y-kh4mell — APPROVED** [review]

**Review by pawelkowalczyksag — APPROVED** [review]

File: content/users-guide/cockpit-bundle/exports.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ The **Time range** filter can filter data for a specific time range. Select a ti
 
 Select the **Object to export** and **Time range** checkboxes to enable the respective filters.
 
->**Info:** The maximum number of documents matching the defined filters that can be returned is 1 million. If the number of documents for the defined filters exceeds this limit, only the first 1 million documents will be exported.
+>**Info:** The maximum number of documents matching the defined filters that can be returned is 1 million. If the number of documents for the defined filters exceeds this limit, only the first 1 million documents will be exported. Additionally, when the result is truncated due to this limitation, an additional line with an indicator is added at the end of the file. The indicator row contains the statements "Limit exceeded!" and "Result truncated!" alternating every other column.
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/exports.md:49]: "Limit exceeded!" > @kaczynskid-sag I slightly changed the formatting (double quotes) amd the spelling (capital letter). Not sure about the second one. Is this spelled with a capital letter in the export file? At least it should :-).

Comment (kaczynskid-sag) [reply content/users-guide/cockpit-bundle/exports.md:49]: No, it is actually spelled with all lower case in the export file. I took it directly from the code here: https://github.softwareag.com/IOTA/cumulocity-core/blob/develop/cumulocity-platform-services/export-management/export-management-service/src/main/java/com/cumulocity/export/impl/export/DocumentExporter.java#L97

**Review by jkot-softwareag — APPROVED** [review]

File: content/users-guide/cockpit-bundle/exports.md (hunk 1 above)
Comment (BeateRixen) [reply content/users-guide/cockpit-bundle/exports.md]: Ok, changed it back for now.

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@kaczynskid-sag Shall I merge this?

**Pull request merged**